src/encodingsettings.html

```diff
@@ -136,6 +136,14 @@ <h3 class="checkboxListLabel">${LabelEnableHardwareDecodingFor}</h3>
                     <div class="fieldDescription">${H264CrfHelp}</div>
                 </div>
 
+                <div class="selectContainer">
+                    <select is="emby-select" id="selectDeinterlaceMethod" label="${LabelDeinterlaceMethod}">
+                        <option value="yadif">Yadif</option>
+                        <option value="yadif_bob">Yadif Bob</option>
```

Comment:
Good point.
Frankly I'm not sure what/how these should be translated. "Yadif" is a program name (originally comes as an acronym from Yet Another Deinterlacing Filter) and "bob" typically indicates interpolation to get 60p (instead of throwing out frames to get 30p)

Reply:
It's mostly if translators prefer to write it in their local script. Like, say, a Japanese translator decides to write it as `ヤディフ` for some reason.

It gives people options if they want them, otherwise it'll use the defautl en_US version.